bound/bound.go

```diff
@@ -16,9 +16,8 @@ func (b Bound) IsValid() bool {
 	return b.Min <= b.Max
 }
 
-// Intersection returns a Bound that is the intersection of the input bounds.
-// If the intersection is empty or if the input length is zero,
-// then the NaN Bound will be returned.
+// Intersection returns a contiguous intersection of the input bounds if possible.
```

Comment:
You can remove "contiguous" since all possible results from the intersection of contiguous intervals will be contiguous or null. Also change "a" to "the" since it returns the only possible answer.